console/console-init/ui/src/modules/iot-device/containers/DeviceListContainer/DeviceListContainer.tsx

```diff
@@ -43,15 +49,18 @@ export interface IDeviceListContainerProps {
   selectedDevices: IDevice[];
   areAllDevicesSelected: boolean;
   selectAllDevices: (devices: IDevice[]) => void;
-  sortValue?: ISortBy;
-  setSortValue: (value: ISortBy) => void;
+  sortValue?: ISortByWrapper;
+  setSortValue: (value: ISortByWrapper) => void;
   appliedFilter: IDeviceFilter;
   resetFilter: () => void;
   projectname: string;
   namespace: string;
-  selectedColumns?: string[];
+  selectedColumns: string[];
   setIsAllSelected: (value: boolean) => void;
 }
+export interface ISortByWrapper extends ISortBy {
```

Comment:
Most of the interfaces being used are not used elsewhere, in any of the other files, therefore we should export only those interfaces that are needed in other components as well.

Reply:
ISortByWrapper is used in different files i.e in quries/iot_device.ts, ManageColumnModal and some other files 
as well